sequence_2/section_2.md

```diff
@@ -1,3 +1,30 @@
+# Section 2: Applications of Redstone Basics
+
+## Piston Doors
+
+This section concerns making doors powered by pistons and redstone. Rather than using wood or iron doors, we'll be using pistons to make large doors using blocks. For this you will need redstone, redstone torches, and sticky pistons. Since sticky pistons can both push and pull blocks, they're ideal for making something like a door. The redstone circuit used is simple but still requires some space, which can be hidden inside a wall (especially since doors normally appear in walls!).
+
+1. The door will be two blocks wide and three blocks tall. Place two columns of three sticky pistons facing each other with four empty air blocks between them. Then place your door blocks (lapiz, gold, etc) onto each of the sticky piston faces. When unpowered, there will be a two-block gap between the pieces of the door.
+![The initial door blocks](images/section_1/piston_door_initial.png)
```

Comment:
These images aren't broken, but should be moved and the references updated for consistency.